### PR TITLE
Add/Fix missing Ids.

### PR DESCRIPTION
**Pull request by Alstruit**
Fixes removed ids due to:
Case sensitive mismatch:
Country origin mismatch:
Updated ID in database but not in streams:
Missing or Removed or moved TV or channel or other broadcasting type next to name.
Romanization/English mismatch.
Re-add ids.

Remove some broken streams.
Move some streams into correct broadcasting origin.
Add some ids.

Unresolved:
streams/ru_okkotv.m3u:
[Terra](https://github.com/iptv-org/iptv/compare/master...Alstruit:iptv2:alstruit-11.8.23?expand=1#diff-263bf28f9360fd196022bbb81623458220a9969ef76f05ec816de910bbacf425L44-R45)

Correct ID, Incorrect Picture:
iptv-org/database#6855
Fix in iptv-org/database#7485.
[Git.ua](https://github.com/iptv-org/iptv/compare/master...Alstruit:iptv2:alstruit-11.8.23?expand=1#diff-f7b82d2550a8cb1eb9735a2d61a5d391c4251b85124829408221d8badee59c79L33-R33)

### CODE REVIEW

File: streams/in.m3u [hunk 1]
```diff
@@ -1,6 +1,4 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="",3 Tamil TV (720p) [Not 24/7]
-https://6n3yogbnd9ok-hls-live.5centscdn.com/threetamil/d0dbe915091d400bd8ee7f27f0791303.sdp/index.m3u8
```
Comment (Dum4G) [inline streams/in.m3u:3]: works totally fine for me. Does it need a removal?
image

File: streams/mx.m3u [hunk 2]
```diff
@@ -127,8 +127,6 @@ https://5ca39be538307.streamlock.net/telemetrika/smil:telemetrika.smil/playlist.
 https://video1.getstreamhosting.com:1936/8172/8172/playlist.m3u8
 #EXTINF:-1 tvg-id="RCGTV2.mx",RCG TV 2 (360p) [Not 24/7]
 https://5caf24a595d94.streamlock.net:1937/stream56/stream56/playlist.m3u8
-#EXTINF:-1 tvg-id="",Romanza+ Africa (720p)
-https://origin3.afxp.telemedia.co.za/PremiumFree/romanza/playlist.m3u8
```
Comment (Dum4G) [inline streams/mx.m3u:131]: works totally fine for me. Does it need a removal?
image

File: streams/pt_samsung.m3u [hunk 3]
```diff
@@ -1,6 +1,4 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="",Africanews
-https://rakuten-africanews-1-pt.samsung.wurl.com/manifest/playlist.m3u8
```
Comment (Dum4G) [inline streams/pt_samsung.m3u:3]: needs .com to be changed for .tv

File: streams/ru_okkotv.m3u [hunk 4]
```diff
@@ -99,25 +99,25 @@ https://okkotv-live.cdnvideo.ru/channel/Russia1HD.m3u8
 https://okkotv-live.cdnvideo.ru/channel/Russia24.m3u8
 #EXTINF:-1 tvg-id="RussiaK.ru",Россия К (480p) [Geo-blocked]
 https://okkotv-live.cdnvideo.ru/channel/Russia_K_SD.m3u8
-#EXTINF:-1 tvg-id="",САПФИР (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id="FoxLifeRussia.ru",САПФИР (1080p) [Geo-blocked]
```
Comment (Dum4G) [inline streams/ru_okkotv.m3u:102]: Needs an update in database to reflect the change

File: streams/ru_okkotv.m3u [hunk 5]
```diff
@@ -99,25 +99,25 @@ https://okkotv-live.cdnvideo.ru/channel/Russia1HD.m3u8
...
 https://okkotv-live.cdnvideo.ru/channel/Fox_Life_HD.m3u8
-#EXTINF:-1 tvg-id="",Солнце (480p) [Geo-blocked]
+#EXTINF:-1 tvg-id="DisneyChannelRussia.ru",Солнце (480p) [Geo-blocked]
```
Comment (Dum4G) [inline streams/ru_okkotv.m3u:104]: Needs an update in database to reflect the change

File: streams/ru_yandex.m3u [hunk 6]
```diff
@@ -1,53 +1,53 @@
...
-#EXTINF:-1 tvg-id="",Bridge Classic (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id="BRIDGEClassic.ru",Bridge Classic (1080p) [Geo-blocked]
 https://strm.yandex.ru/kal/bridgetvclassic/bridgetvclassic0.m3u8
-#EXTINF:-1 tvg-id="",Bridge Deluxe (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id="BRIDGEDeluxe.ru",Bridge Deluxe (1080p) [Geo-blocked]
 https://strm.yandex.ru/kal/bridgetvdeluxe/bridgetvdeluxe0.m3u8
-#EXTINF:-1 tvg-id="",Bridge Hits (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id="BRIDGEHits.ru",Bridge Hits (1080p) [Geo-blocked]
 https://strm.yandex.ru/kal/bridgetvhits/bridgetvhits0.m3u8
 #EXTINF:-1 tvg-id="",Bridge Rock (1080p) [Geo-blocked]
```
Comment (Dum4G) [inline streams/ru_yandex.m3u:14]: Isn't addressed yet

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: streams/ru_okkotv.m3u [hunk 7]
```diff
@@ -21,51 +21,51 @@ https://okkotv-live.cdnvideo.ru/channel/Amedia_Hit_HD.m3u8
...
-#EXTINF:-1 tvg-id="",Star Family (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id="StarFamily.ru",Star Family (1080p) [Geo-blocked]
 https://okkotv-live.cdnvideo.ru/channel/Star_Family_HD.m3u8
-#EXTINF:-1 tvg-id="",START Air (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id="STARTAir.ru",START Air (1080p) [Geo-blocked]
 https://okkotv-live.cdnvideo.ru/channel/START_Air_HD.m3u8
-#EXTINF:-1 tvg-id="",START World (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id="STARTWorld.ru",START World (1080p) [Geo-blocked]
 https://okkotv-live.cdnvideo.ru/channel/START_World_HD.m3u8
 #EXTINF:-1 tvg-id="",TERRA (1080p) [Geo-blocked]
```
Comment (Dum4G) [inline streams/ru_okkotv.m3u:44]:
```suggestion
#EXTINF:-1 tvg-id="TERRA.ru",TERRA (1080p) [Geo-blocked]
```

**Review by Dum4G — CHANGES_REQUESTED** [review]

**Comment by Dum4G** [comment]
That's dope. Have you used some automation script or did it all manually?

File: streams/in.m3u (hunk 1 above)
Comment (Alstruit) [reply streams/in.m3u:3]: [I moved it here.](https://github.com/iptv-org/iptv/pull/14997/files#diff-17ca22b929d4575883d9d9936826bcaaa700f0eb18bf9ae0eaa0c4ef52d5178aR6-R7)

File: streams/mx.m3u (hunk 2 above)
Comment (Alstruit) [reply streams/mx.m3u:131]: [I moved it here.](https://github.com/iptv-org/iptv/pull/14997/files#diff-c89720c3c1bb8c4172c3f0da66547995d2092bd7b0f12dc400edaa9b18da60b0R50-R51)

File: streams/pt_samsung.m3u (hunk 3 above)
Comment (Alstruit) [reply streams/pt_samsung.m3u:3]: Will be resolved in a different PR.

File: streams/ru_okkotv.m3u (hunk 7 above)
Comment (Alstruit) [reply streams/ru_okkotv.m3u:44]: Not in [database.](https://iptv-org.github.io/?q=TERRA.ru)

File: streams/ru_okkotv.m3u (hunk 5 above)
Comment (Dum4G) [inline streams/ru_okkotv.m3u:104]:
```suggestion
#EXTINF:-1 tvg-id="Solnce.ru",Солнце (480p) [Geo-blocked]
```

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: streams/mx.m3u (hunk 2 above)
Comment (Dum4G) [reply streams/mx.m3u:131]: belongs to TV Azteca Networks ([TV Azteca](https://en.wikipedia.org/wiki/TV_Azteca)) from Mexico

File: streams/ru_yandex.m3u (hunk 6 above)
Comment (Dum4G) [inline streams/ru_yandex.m3u:14]:
```suggestion
#EXTINF:-1 tvg-id="BRIDGERock.ru",Bridge Rock (1080p) [Geo-blocked]
```

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: streams/pt_samsung.m3u (hunk 3 above)
Comment (Alstruit) [reply streams/pt_samsung.m3u:3]: #14998

**Comment by Dum4G** [comment]
Have you used some automation script or did it all manually?

File: streams/mx.m3u (hunk 2 above)
Comment (Alstruit) [reply streams/mx.m3u:131]: Done moved it back

**Review by Dum4G — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**